throw.go

```diff
@@ -82,16 +84,20 @@ func doThrow(c *cli.Context) error {
 			return client.PostHostMetricValuesByHostID(optHostID, metricValues)
 		}, optMaxRetry))
 
-		for _, metric := range metricValues {
-			logger.Log("thrown", fmt.Sprintf("%s '%s\t%f\t%d'", optHostID, metric.Name, metric.Value, metric.Time))
+		if !optVerbose {
+			for _, metric := range metricValues {
```

Comment:
Well, I know this is kind of like the hell, but `logger` has no configuration about log level.